include/glow/ExecutionEngine/ExecutionEngine.h

```diff
@@ -96,7 +96,8 @@ class ExecutionEngine final {
   /// without resetting the backend.
   void compile(CompilationContext &cctx);
 
-  /// A convenience function for the most common type of compile.
+  /// A convenience function for the most common type of compile. Can only be
```

Comment:
can you also update getModule() comment. It's not safe to call it after graph has been compiled.

Reply:
Updated the comment to call out that elements of the module may no longer exist. The module itself is safe to access after compile though.